ethdb/database.go

```diff
@@ -158,7 +158,7 @@ type FreezerEnv struct {
 // AncientFreezer defines the help functions for freezing ancient data
 type AncientFreezer interface {
 	// SetupFreezerEnv provides params.ChainConfig for checking hark forks, like isCancun.
-	SetupFreezerEnv(env *FreezerEnv) error
```

Comment:
blockHistory is similar to BlobExtraReserve?
if so, maybe can add it as a field of FreezerEnv